Docs/source/usage/parameters.rst

```diff
@@ -21,12 +21,12 @@ Overall simulation parameters
     When provided, this information is added as metadata to (openPMD) output files.
 
 * ``max_step`` (`integer`)
-    The number of PIC cycles to perform.
+    The number of PIC cycles to perform. See :ref:`stopping the simulation <stopping-the-simulation>`.
 
 * ``stop_time`` (`float`; in seconds)
     The maximum physical time of the simulation. Can be provided instead of ``max_step``. If both
```

Comment:
```suggestion
    The maximum physical (in the boosted frame if doing a boosted-frame calculation) time of the simulation. Can be provided instead of ``max_step``. If both
```

Reply:
This is redundant but I think is worth pointing out.

Reply:
I'll add this, but with a change in the wording.